### ISSUE TITLE
daemon's ames i/o driver should print less aggressively

### ISSUE THREAD

**Issue by custom-jonathan** [urbit/urbit#3407]
**Describe the bug**
I don't know what it means, but I'm seeing this in my dojo. Doesn't seem to be affecting anything.

![image](https://user-images.githubusercontent.com/51895363/91675644-b194c380-eb02-11ea-938b-a0f3e98ad16c.png)

**Comment by belisarius222** [comment]
It looks like an intermittent internet connection to me.  The daemon's Ames I/O driver should print less aggressively, though, like how packet forwards are only printed once every n packets.

**Comment by matildepark** [comment]
Revised a bit — trying to keep these well-focused to the exact site of remedy

**Comment by joemfb** [comment]
The "send fail" printf is new in `v0.10.8` -- we never used to print those errors. I'd be fine with just removing it, but a heuristic, periodic message would be best.

**Comment by custom-jonathan** [comment]
I tried to |hi ~roswet which is the star I'm syncing to, and it hasn't gone through nor did the OTA from last night land on ~minder-folden. Still getting this printout pretty consistently and on the regular. Any chance it's connected?

**Comment by custom-jonathan** [comment]
![image](https://user-images.githubusercontent.com/51895363/91988688-2d566200-ecf5-11ea-91dc-f2542f31c50e.png)

**Comment by custom-jonathan** [comment]
And this counter keeps heading north: `ames: packet dropped (196000 total)` 

Don't know if it's connected or not, but figured I'd put it here.

**Comment by joemfb** [comment]
@custom-jonathan, those `czar` printfs mean you aren't able to resolve galaxy domains names, which suggests that your internet (or at least your DNS) is down, or misconfigured.

The dropped packets, on the other hand, suggest degraded performance. Do you have swap configured on this VM, by any chance?

**Comment by custom-jonathan** [comment]
I think I do have swap on here. I followed the Medium article written by the Urbit Live crew. I also increased my droplet size recently, too. Restarted the droplet and it seemed to be able to resolve the galaxies. Still not getting the ota, and some messages in the general urbit help channels are taking upwards of an hour.

**Comment by custom-jonathan** [comment]
Realized while I increased the harddrive size I was still at 2GB on RAM, so I upgraded to 4GB of RAM and powered it back up. Ran `|reset` then `|pack` and still getting the ames dropped packets message.

**Comment by custom-jonathan** [comment]
Noticed this for the first time, too:
```
error in %hood
%arvo-response
! /lib/hood/kiln/hoon:<[242 5].[253 7]>
! /lib/hood/kiln/hoon:<[241 5].[253 7]>
! /lib/hood/kiln/hoon:<[472 30].[472 60]>
! /lib/hood/kiln/hoon:<[472 25].[472 60]>
! /lib/hood/kiln/hoon:<[463 3].[481 5]>
! /app/hood/hoon:<[113 32].[113 64]>
! /app/hood/hoon:<[113 16].[113 74]>
! /app/hood/hoon:<[110 3].[114 5]>
! /app/hood/hoon:<[109 3].[114 5]>
```

**Comment by joemfb** [comment]
@custom-jonathan, thanks for ruling out swap.

I'm not sure what might be causing these performance problems. Dropping packets like that is really unusual, on anything but a highly-trafficked galaxy. Could you enable some %ames debugging output and post it here (or to a pastebin, if it's a lot)?

```
> |ames-sift
> |ames-verb %rcv %for %snd %msg
```

after, just run this to disable that output:

```
> |ames-verb
```

/cc @philipcmonk

**Comment by joemfb** [comment]
The titular issue here (ames error printf frequency) is fixed in #3693.

**Comment by joemfb** [comment]
Fix released.
